.changeset/dull-flowers-build.md

```diff
@@ -2,16 +2,13 @@
 'skuba': minor
 ---
 
-lint: Update Docker base images to point to AWS ECR Public and remove redundant `--platform` usage
+format, lint: Point Docker base images to AWS ECR Public and remove constant `--platform` arguments
 
-This updates references to `node:` or `python:` Docker images in your `Dockerfile` and `docker-compose.yml` files to point to AWS ECR Public to avoid Docker Hub rate limiting, along with removing redundant `--platform` [usage](https://docs.docker.com/reference/build-checks/from-platform-flag-const-disallowed/).
+This updates references to `node:` or `python:` Docker images in your Dockerfiles and `docker-compose.yml` files to point to AWS ECR Public to avoid Docker Hub rate limiting. It also removes [constant `--platform` arguments](https://docs.docker.com/reference/build-checks/from-platform-flag-const-disallowed/) from Dockerfiles.
 
-eg.
-
-```Dockerfile
-## Before
-FROM --platform=arm64 node:20-alpine AS dev-deps
-
-## After
-FROM public.ecr.aws/docker/library/node:20-alpine AS dev-deps
+```diff
+- FROM --platform=arm64 node:20-alpine AS dev-deps
++ FROM public.ecr.aws/docker/library/node:20-alpine AS dev-deps
 ```
+
+Your Dockerfiles may not be set up to build multi-platform images, so keep in mind that building them locally on an Intel x86 laptop may not yield images that can execute on AWS Graviton instances.
```

Comment:
I think this sad disclaimer is necessary? Kinda made me question if we should follow the last example here and have `_arm64` stage names instead: https://docs.docker.com/reference/build-checks/from-platform-flag-const-disallowed/

Reply:
🤷